src/server/index.tsx

```diff
@@ -33,16 +33,20 @@ const app: express.Express = express();
 const rawBase: string = process.env.BASE_PATH || '';
 // remove trailing slash
 const base: string = rawBase.replace(/\/$/, '');
+// Utility fonction to compute the full path depending on the base path
+const buildPath = buildPathFactory(base);
+// base path ending with a slash, for routing purpose
+const fullBasePath = buildPath();
 // enable logs
 app.use(morgan('dev'));
 // expose status route
-app.get(`${base}/status`, (req, res) => res.send('OK'));
+app.get(buildPath('status'), (req, res) => res.send('OK'));
```

Comment:
Hmm, not sure... We're actually using path the wrong way. We should create normal express routers and mount it at `${base}` if there's one.

Also, we need to re-configure the service at openshift level